packages/styles/global/variables.scss

```diff
@@ -70,4 +70,7 @@
   --zindex-1000: 1000;
   --zindex-overlay: 10000;
   --transition-duration: 0.25s;
+  --font-weight-regular: 400;
+  --font-weight-medium: 500;
+  --font-weight-bold: 900;
```

Comment:
@schirrel could we add this informations inside storybook?

Reply:
Sure, thanks for remember me, can be add at `2_Typography.stories.mdx` file?

Reply:
Yes! It will works